guide/english/certifications/javascript-algorithms-and-data-structures/es6/mutate-an-array-declared-with-const/index.md

```diff
@@ -44,6 +45,21 @@ Reassign the values of the `const` variable `s` using various element assignment
 
 Trying to reassign a read-only `const` variable will throw an error, but by using various element assignment you can access and change the value of an array just like you would with `let` or `var`.
 
+## ![:beginner:](https://forum.freecodecamp.com/images/emoji/emoji_one/beginner.png?v=3 ":beginner:") Advanced Code Solution:
+```javascript
+    const s = [5, 7, 2, 8];
+    function editInPlace() {
+      "use strict";
+      s.unshift(s.pop());
+    }
+    editInPlace();
+```
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://codepen.io/imambungo/pen/rQjXpX' target='_blank' rel='nofollow'>Run Code</a>
```

Comment:
Please remove this line as we no longer want external links like this to personal accounts.
```suggestion
```